src/main/java/org/jenkinsci/plugins/matrixauth/AuthorizationContainerDescriptor.java

```diff
@@ -132,7 +132,7 @@ default PermissionEntry entryFor(String type, String sid) {
     @Restricted(DoNotUse.class) // Jelly only; cf. UpdateCenter#getCategoryDisplayName in core
     default String getTypeLabel(String type) throws NoSuchMethodException, InvocationTargetException, IllegalAccessException {
         if (type == null) {
-            return null;
+            return "__TYPE__";
```

Comment:
new values were showing null before after clicking add, this isn't fully fixed as it's not localised yet, but I thought I'd push it for now